CONTRIBUTING.md

```diff
@@ -39,6 +40,8 @@ This project was generated by the [dash-component-boilerplate](https://github.co
         ```
         $ npm run build
         ```
+        **N.B.**
+        + We use `postbuild_fixups.sh` to re-write certain lines in the generated Julia files of this package to allow usage of Julia's [Artifact](https://pkgdocs.julialang.org/v1/artifacts/#Artifacts) system.
```

Comment:
This is fine, we do need to run `npm run postbuild` while developing the package.

But more importantly, we should also mention `npm run postbuild` in the [Create a production build and publish](https://github.com/etpinard/dash-textarea-autocomplete/blob/f6ede405e6ab3fe4a1b29f8c4c2dc1bc9268f771/CONTRIBUTING.md#create-a-production-build-and-publish) section below.

Reply:
From what I've been doing, `npm run build` seems to run every step of the `"scripts"` section in `package.json`, regardless of the name as long as the script entry starts with `npm run ...`  

I've haven't had to do `npm run postbuild` yet to have the `postbuild:<>` step run so far.

Maybe it's different for a production build though ?

Reply:
Ah I see. Looks like the `pre` and `post` complementary scripts work now even for user-defined scripts like `build`.

- https://docs.npmjs.com/cli/v6/using-npm/scripts
- https://thomasxbanks.com/articles/pre-post-npm-scripts/